packages/fsweb/polyfills/navigator.js

```diff
@@ -1 +1,2 @@
 export const languages = [];
+export const userAgent = 'na';
```

Comment:
Some of the `material-ui` components crash unless we mock the `userAgent` that has some value.